.github/workflows/cicd.yml

```diff
@@ -150,6 +150,25 @@ jobs:
       # driver tests manually gather code coverage
       code_coverage: false
 
+  # Run the driver tests on self-hosted runners.
+  driver_ws2022:
+    # Always run this job.
+    # Only run this on repos that have self-host runners.
+    needs: regular
+    if: github.repository == 'microsoft/ebpf-for-windows'
```

Comment:
why is the logic here different than line 139?

Reply:
bad merge after Alan added the restriction recently